topsort-api-v1.yml

```diff
@@ -264,6 +264,7 @@ components:
           type: number
           minimum: 0
           maximum: 1
+          exclusiveMaximum: true
```

Comment:
I thought the change they made yesterday allowed the qualityScore to be 1, so this should not exclusive.